hypervisor/src/kvm/mod.rs

```diff
@@ -602,10 +602,9 @@ impl vm::Vm for KvmVm {
             .map_err(|e| vm::HypervisorVmError::SetGsiRouting(e.into()))
     }
 
-    ///
+    /// # Safety
     /// Creates a memory region structure that can be used with {create/remove}_user_memory_region
```

Comment:
A safety doc comment is supposed to explain the invariants the function requires users to ensure — not just to explain what the function does, which should go above the safety comment.